scraper.py

```diff
@@ -0,0 +1,102 @@
+import json
+
+import requests
+import bs4
+from bs4 import BeautifulSoup, NavigableString, Tag
+
+
+# function to get URLs from page
+def get_url_title(url):
+    page = requests.get(url)
+    soup = BeautifulSoup(page.text, 'html.parser')
+    # finds all links to API/API Standards as listed on the page
+    links = [a['href'] for a in soup.find_all('a', href=True)]
+    filtereddivs = list(filter(lambda x: '/developer/api-catalogue/' in x, links))
```

Comment:
in Python the convention is to separate words in variables names with an underscore, so this would be filtered_divs